docs/spec-files/problems.md

```diff
@@ -164,10 +164,10 @@ To troubleshoot routing issues, DIDComm offers a header, `trace`. Any party that
 
 ```json
 {
-  "type": "https://didcomm.org/trace/1.0/trace_report",
+  "type": "https://didcomm.org/trace/2.0/trace_report",
```

Comment:
this is fuzzy as it's not a defined protocol name. I think leaving 2.0 here is the right answer.